ecosystem/sep-0010.md

```diff
@@ -36,6 +36,10 @@ The authentication flow is as follows:
    1. Source account set to the user's account.
    1. Key set to `<home domain> auth` where the home domain is the home domain from the discovery flow.
    1. Value set to a nonce value.
+1. The client verifies that if the transaction has a Manage Data operation with key `server_domain` that it has:
```

Comment:
I'm a bit concerned that `server_domain` is too vague. What about `sep10_server_domain`?

Reply:
I think we should keep the term `SEP-10` out of the code interface, SEPs do not normally include that. How about `issuer_domain`? Using this name will do one of two things:
- Clear up misconception because the issuer of the SEP-10 challenge is the SEP-10 server.
- Any ambiguity should be more apparent since the term `issuer` may more likely lead to the reader digging in to find out what it means rather than glossing over on a generic term such as `server`.

Reply:
I think `server_domain` is a better than `issuer_domain` due to the fact that the term `issuer` is already a keyword within the Stellar context and could confuse the reader. What about `web_auth_server`, which would be the domain used in SEP-1's `WEB_AUTH_ENDPOINT`?

Reply:
Oh I like `web_auth_domain`. It correlates directly to this SEP. I like it. Updated. 01a77b4

Reply:
One more thought on this though, we use the term `server_account`  to mean the account of the SEP-10 server, so if we continued to use the term `server_domain` it correlates.

Reply:
Ah thats true, that definitely makes it more plausible. I still think `web_auth_domain` is better though.